src/components/Articles.jsx

```diff
@@ -1,10 +1,36 @@
+import { getArticleById } from "../utils/api";
 import ArticleCard from "./ArticleCard";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
 
 const Articles = ({ articles }) => {
-    console.log(articles)
+  const [input, setInput] = useState(0);
+  const navigate = useNavigate();
+
+  const updateInput = (e) => {
+    setInput(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+    console.log(e);
+    navigate(`/articles/${input}`);
+  };
+
   return (
     <>
       <h2>Articles</h2>
+      <form onSubmit={handleSubmit}>
+        <label>
+          Search Article ID:{" "}
+          <input
+            type="number"
+            placeholder="Article ID"
+            value={input}
+            onChange={updateInput}
+          ></input>
+        </label>
+        <button id="search-article-btn">Search</button>
+      </form>
```

Comment:
The article ID means nothing to the user - they don't know which ID is which article. It would be much better to let them get to the article by linking to it from the homepage